src/Compilers/CSharp/Portable/BoundTree/Constructors.cs

```diff
@@ -244,7 +245,8 @@ public static BoundConversion SynthesizedNonUserDefined(SyntaxNode syntax, Bound
             bool explicitCastInCode,
             ConstantValue constantValueOpt,
             TypeSymbol type,
-            bool hasErrors = false)
+            bool hasErrors = false,
+            bool? isNullable = null)
```

Comment:
>bool? isNullable = null [](start = 12, length = 23)

I think providing default value for this parameter prevents us from making sure that all call sites are properly adjusted.